### ISSUE TITLE
The action build_windows in ytdl-patched is not working since 2024.05.10.1919

### ISSUE THREAD

**Issue by janoi152** [ytdl-org/youtube-dl#32846]
## Checklist

- [x] I'm reporting a broken site support issue
- [x] I've verified that I'm running youtube-dl version **2021.12.17**
- [x] I've checked that all provided URLs are alive and playable in a browser
- [x] I've checked that all URLs and arguments with special characters are properly quoted or escaped
- [x] I've searched the bugtracker for similar bug reports including closed ones
- [x] I've read bugs section in FAQ

## Verbose log

```
Run python -m pip install --upgrade "pip<19.2" "setuptools<44" "wheel<0.34.0" py2exe==0.9.2.2
  python -m pip install --upgrade "pip<19.[2](https://github.com/ytdl-patched/youtube-dl/actions/runs/9867247587/job/27247422628#step:4:2)" "setuptools<44" "wheel<0.34.0" py2exe==0.9.2.2
  shell: C:\Program Files\PowerShell\7\pwsh.EXE -command ". '{0}'"
  env:
    pythonLocation: C:\hostedtoolcache\windows\Python\[3](https://github.com/ytdl-patched/youtube-dl/actions/runs/9867247587/job/27247422628#step:4:3).4.4\x86
Requirement already up-to-date: pip<19.2 in c:\hostedtoolcache\windows\python\3.[4](https://github.com/ytdl-patched/youtube-dl/actions/runs/9867247587/job/27247422628#step:4:4).4\x86\lib\site-packages
Requirement already up-to-date: setuptools<44 in c:\hostedtoolcache\windows\python\3.4.4\x8[6](https://github.com/ytdl-patched/youtube-dl/actions/runs/9867247587/job/27247422628#step:4:7)\lib\site-packages
Collecting wheel<0.34.0
  Could not find a version that satisfies the requirement wheel<0.34.0 (from versions: )
No matching distribution found for wheel<0.34.0
Error: Process completed with exit code 1.
```

## Description

In ytdl-patched fork since release 2024.05.10.1919 a Windows executable is no longer being created. The action build_windows keeps failing. Can someone please fix it? Thank you.

**Comment by dirkf** [comment]
Try the Windows executable in the nightly build here (tomorrow, once PR #32845 is merged). I believe that ytdl-patched is no longer maintained, maybe because we now have the nightly build here.

Please report on any feature that you think is missing compared with the ytdl-patched build.

**Comment by dirkf** [comment]
As you posted here, I assumed you meant ytdl-patched/youtube-dl, which is completely superseded by the nightly releases here, at ytdl-org//ytdl-nightly, and has not been updating since June 2003.

There is also ytdl-patched/ytdl-patched, which is a fork of _yt-dlp_ with additional patches. This likewise has not been updating from its parent for many months.

AFAIK the developer responsible for the ytdl-patched repo is no longer active. If you want a site or feature supported by ytdl-patched/ytdl-patched that has not been included in _youtube_dl_ or_yt-dlp_, you should raise an issue in the appropriate issue tracker.
